packages/decorators/src/djs-decorators.ts

```diff
@@ -0,0 +1,121 @@
+import { isDMChannel, isGuildBasedChannel } from '@sapphire/discord.js-utilities';
+import { UserError } from '@sapphire/framework';
+import { Message, PermissionResolvable, Permissions } from 'discord.js';
+import { createFunctionPrecondition, FunctionFallback } from './utils';
+
+export const enum DecoratorIdentifiers {
+	RequiresPermissionsGuildOnly = 'requiresPermissionsGuildOnly',
+	RequiresPermissionsMissingPermissions = 'requiresPermissionsMissingPermissions'
+}
+
+// TODO: Re-enable commented out permissions when updating to DJS v13
+const ServerOnlyPermissions = new Permissions([
```

Comment:
Instead of doing this, you can negate it by specifying the DM-only permissions, and then use the `~` unary operator to flip the bits and get the non-DM permissions.

`~0b11000111` ===
` 0b00111000`